src/components/Navigation.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import 'bulma/css/bulma.min.css';
+import { Button } from 'react-bulma-components';
+
+export const Navigation: React.FC = () => {
+  return (
+    <nav className="App__nav">
+      <NavLink
+        to="/home"
+        className={({ isActive }) => (isActive ? 'is-active' : '')}
```

Comment:
I believe using `classnames` would be a better solution instead of applying ternary with an empty string